jobs/build.Jenkinsfile

```diff
@@ -308,8 +308,9 @@ lock(resource: "build-${params.STREAM}") {
             shwrap("""
             cosa generate-hashlist --arch=${basearch} --release=${newBuildID} \
                 --output=builds/${newBuildID}/${basearch}/exp-hash.json
-            sha256sum builds/${newBuildID}/${basearch}/exp-hash.json \
-                > builds/${newBuildID}/${basearch}/exp-hash.json-CHECKSUM
+            source="builds/${newBuildID}/${basearch}/exp-hash.json"
+            target="builds/${newBuildID}/${basearch}/exp-hash.json-CHECKSUM"
+            cosa shell -- bash -c "sha256sum \$source > \$target"
```

Comment:
I can imagine this `cosa shell` trap being something we easily forget and regress on. An alternative is introduce a new e.g. `cosa_shwrap()` which is like `shwrap` but automatically runs on the remote host via `cosa shell` in the remote invocation case.

Reply:
i.e. we easily forget when adding new code to do the right thing and appropriately use `cosa shell -- cmd` when needed? That is a valid concern. I'm not in love with the idea of a `cosa_shwrap()` as I don't really think it alleviates the concern; we can still regress easily. I think it's just something we need to be aware of.. and possibly more aggressively refactor and deduplicate similar code in the future.